environments/staging/meta.yml

```diff
@@ -7,5 +7,6 @@ users:
   - g2
   - fissionlabs
   - fission-warehouse
+  - staging
```

Comment:
I don't understand why `staging` would be added here. Should this be `frener`?

Reply:
Hi Daniel, because I add frener in [staging.yml](https://github.com/dimagi/commcare-cloud/blob/136b671ef02f07e3d9e05753c08f826fe43b31ea/environments/_users/staging.yml)

Reply:
Oh, I see, this is a list of yaml files (without `.yml` extensions), not actual users. Makes sense now.